components/Project.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+export default function Project({ title, duration, complexity }) {
+  return (
+    <StyledArticle>
+      <h3>{title}</h3>
+      <p>Duration: {duration}</p>
+      <p>Complexity: {complexity}</p>
+    </StyledArticle>
+  );
+}
+
+const StyledArticle = styled.article`
+  border: 1px solid black;
+  margin: 10px;
+  padding: 0px 0px 0px 10px;
+  width: 90vw;
+`;
```

Comment:
The parent element is already a flex container, therefore you can simplify `StyledArticle` component by relying on the flexbox layout defined by the parent. 

- If the flex container uses the `gap` property, you can remove the `margin` from the StyledArticle child elements and still maintain the desired spacing between them

_Here's how you can simplify it, but please note that this code is written "dry" and not tested._

```suggestion
const StyledArticle = styled.article`
  border: 1px solid black;
  padding: 0px 10px;
  max-width: 90vw;
```

Reply:
Done